### ISSUE TITLE
Estimated excess ILI is too high

### ISSUE THREAD

**Issue by wmay** [jsilve24/ili_surge#2]
To estimate excess non-influenza ILI, you're using a method that predicts winter 2019 levels based on 2010-July 2019 levels. But it's clear from figure 1 that the winter 2019 levels are substantially higher than the predictions in many states.

It seems clear that you could get more accurate predictions starting in March 2020 by incorporating the data from previous weeks (July 2019-March 2020) into the prediction. I think this would lead to significantly lower excess ILI estimates, higher case detection rate estimates, and smaller confidence intervals.

After looking for ways to forecast ILI I realized that the FluSight Network already does this, and they post their data on github [here](https://github.com/FluSightNetwork/cdc-flusight-ensemble).

Unfortunately they don't make state-level forecasts, but if you're willing to switch to a region-level analysis I think it would be straightforward to plug in the FluSight forecasts and update the paper results. It also took me a while to find but I think a lot of the code behind the forecasts is [here](https://github.com/reichlab/2017-2018-cdc-flu-contest) and [here](https://github.com/cmu-delphi/epiforecast-R). So you could potentially make a few changes to get state-level forecasts out of their models.

I'd be wiling to submit a pull request implementing this if you're interested.

**Comment by jsilve24** [comment]
Thanks for the offer and the references. 

We thought a lot about this. Yes, we do see an elevated winter baseline in Montana and Colorado (those are the main two I remember at least). However it was not so clear to us how to handle this. We did consider readjusting baselines but here's the issue: Consider instead the peak in November in Washington. This very clearly lines up with RSV (https://www.cdc.gov/surveillance/nrevss/rsv/state.html#WA <https://www.cdc.gov/surveillance/nrevss/rsv/state.html#WA>). Just readjusting our baseline could give a falsely low estimate. If there was a clear policy or reporting standards change  that would lead to over/under reporting we would of course try to account for this by readjusting the baseline based on the time immediately after/before the change. However, we are not aware of any such change and I worry that just randomly readjusting the baseline because "it looks like it should be adjusted" may actually be a step in the wrong direction. 

Please do not take this as me not listening/being open. I am very interested in this discussion as it is something that has bothered me. That said, It is not so clear to me what the correct choice is here and I am skeptical of the readjusted baseline.

Just to clarify -- we do not expect nor claim thet any excess ILI before march 8th can be attributed to COVID. Is it possible that in Colorado and Montana there was a summer illness going around? From what I have heard this year was a particularly bad flu year. I have been told that the major circulating strain were poorly covered by this years flu shot. So again, I don't know what is truly signal and what might be altered baseline. If you know of distinct policy / reporting changes that would make this a much clearer issue. 
 

Justin

**Comment by jsilve24** [comment]
Just to be clear, regarding my comment on flu. What you are looking at is the signal with the flu removed. However, it is possible that some flu remains in the signal if there is biases in who gets a flu test. Still, that bias should only really effect early march as by mid march influenza seems to have become negligible based on the laboratory testing data/other reports I have seen. 

Justin

**Comment by wmay** [comment]
I appreciate the quick response!

This doesn't seem like a random readjustment to me. (And I'm not sure I'm being totally clear either, but I'm trying.)

I'm a statistician, so I'm looking at this from a purely statistical point of view. Based on your figure 1, it looks clear that higher values early in the season are associated with higher values later on in the season. So when we see high values, say, in January, our expectation of the values in March should go up.

Your current approach doesn't use this information, so it's predictions aren't as precise as they could be. The results could vary a lot depending on whether we're in a good or bad non-influenza ILI season.

If you use a different type of forecasting model that looks at the recent ILI trajectory (the FluSight ones are good examples), you should be able to remove a potentially large source of error, because that year-to-year variation is largely included in the current season's trajectory.

To put it another way, if I wanted to forecast ILI in March, I would definitely look at the ILI values from February, and I don't think it would be arbitrary to do that.

**Comment by jsilve24** [comment]
I agree that this is a point worthy of debate. However, while I am in fact a statistician as well, I tend to dislike the tendency to model the data rather than modeling the underlying system (https://www.biorxiv.org/content/10.1101/477794v1.article-metrics <https://www.biorxiv.org/content/10.1101/477794v1.article-metrics>). 

I think there are valid hypotheses for why your observations could be due to biology rather than random chance or policy changes. However, if there was evidence of policy / reporting changes in various states then I would surely try to account for them. 

That said, I am hesitant to believe that, even if policy/reporting changes were behind what you are highlighting they would change our estimates by an order of magnitude. At this point, order of magnitude is really what I think would be necessary to undermine / substantially refine the key point in the paper. (open to disagreement here thought). 

Justin

**Comment by wmay** [comment]
I guess I'll agree to disagree.

On a related note, I was confused by the case detection rate credible intervals in the April 14th version of the paper. It says,

> In the last week ending on March 28, we estimate the detection rate across the US increased to be 1.5% (95% credible interval 1.2%-1.9%).

But based on the uncertainty in the ILI predictions, I expected a larger interval. I looked through the code and found that, to get total excess ILI intervals, you're taking the sum of the state excesses ([here](https://github.com/jsilve24/ili_surge/blob/3614cb32c8678118506e835c94a0c22aebda761e/R/excess_ili_by_state.R#L796)), transforming that value, and then getting credible intervals of the sum ([here](https://github.com/jsilve24/ili_surge/blob/3614cb32c8678118506e835c94a0c22aebda761e/R/excess_ili_by_state.R#L833)). But this tacitly assumes that the state values are independent of each other, and we know that isn't the case. I couldn't tell if that's where the detection rate credible interval came from, but at least that part is not accounting for state ILI covariances.
